berserk/clients.py

```diff
@@ -521,6 +526,7 @@ def get_ongoing(self, count=10):
         params = {'nb': count}
         return self._r.get(path, params=params)['nowPlaying']
 
+    @deprecated(version='0.12.0', reason='use TV.get_current_games')
```

Comment:
I'm not sure what version you're going to release with next so please let me know if this should be adjusted.